### PR TITLE
Feature 1820 added modal pop up when personal button is press on the onboard progress detail page

### PR DESCRIPTION
**Pull request by BenVuong**
added a basic modal when the personal information button is clicked on the onboard progress detail page

### CODE REVIEW

File: web-ui/src/components/accordion/Accordion.css [hunk 1]
```diff
@@ -0,0 +1 @@
+idth100Percent { width: 100%; }
```
Comment (MatthewBackes) [inline web-ui/src/components/accordion/Accordion.css:1]: Should be width100Percent

**Review by MatthewBackes — APPROVED** [review]

**Comment by S78901** [comment]
We need to use MUI's modal if we can. It's fairly straightforward.

https://mui.com/material-ui/react-modal/

Can you adjust to use this instead of custom code, as Michael Kimberlin suggested?

File: web-ui/src/pages/OnboardProgressDetailPage.jsx [hunk 2]
```diff
@@ -6,10 +6,23 @@ import { DataGrid } from "@mui/x-data-grid";
 import "./../components/modal/Modal";
 import getDocuments from "../api/signrequest";
 import "./OnboardProgressDetailPage.css";
-import Modal from "./../components/modal/Modal";
+//import Modal from "./../components/modal/Modal";
```
Comment (S78901) [inline web-ui/src/pages/OnboardProgressDetailPage.jsx:9]: Let's delete this reference.

File: web-ui/src/components/modal/Modal.css [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.modal{
```
Comment (S78901) [inline web-ui/src/components/modal/Modal.css:1]: We can delete all files for the custom modal too.

File: web-ui/src/components/modal/Modal.jsx [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import React from "react";
```
Comment (S78901) [inline web-ui/src/components/modal/Modal.jsx:1]: This file can be deleted.

**Review by S78901 — DISMISSED** [review]
Great work.

**Review by MatthewBackes — APPROVED** [review]

**Review by li-brandon — APPROVED** [review]

**Pull request merged**